File: android2_lesson01/src/java/java/com/example/ama/android2_lesson01/ui/rv/mvp/ListOfNotesView.java
package com.example.ama.android2_lesson01.ui.rv.mvp;

import com.example.ama.android2_lesson01.ui.base.NoteView;
import com.example.ama.android2_lesson01.model.Note;

import java.util.ArrayList;

/**
 * Interface for work with {@link ListOfNotesPresenter}
 */
public interface ListOfNotesView extends NoteView {
    void showNoteList(ArrayList<Note> mData);

    void showEmptyMessage();

    void hideEmptyMessage();
}

File: android2_lesson01/src/kotlin/java/com/example/ama/android2_lesson01/ui/rv/mvp/ListOfNotesView.kt
package com.example.ama.android2_lesson01.ui.rv.mvp

import com.example.ama.android2_lesson01.model.Note
import com.example.ama.android2_lesson01.ui.base.NoteView

/**
 * Interface for work with [ListOfNotesPresenter]
 */

interface ListOfNotesView : NoteView {
    fun showNoteList(mData: ArrayList<Note>)
    fun showEmptyMessage()
    fun hideEmptyMessage()
}